main/main.cpp

```diff
@@ -581,12 +581,11 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	GLOBAL_DEF("memory/multithread/thread_rid_pool_prealloc", 60);
 
 	GLOBAL_DEF("network/debug/max_remote_stdout_chars_per_second", 2048);
-	GLOBAL_DEF("network/debug/remote_port", 6007);
```

Comment:
I think you need to keep that one to get it registered in the project settings. Here's what I have with your PR:
![spectacle j17470](https://user-images.githubusercontent.com/4701338/27009482-1a1d1fb4-4e90-11e7-9345-282a23985bee.png)